src/types.ts

```diff
@@ -20,6 +20,8 @@ type outcome =
 
 export interface Assertion {
   '@type': EarlType.Assertion
+  'doap:name': string
+  'doap:vendor': Object
```

Comment:
Same as above, these are not valid properties of an Assertion. You need to add these to an Assertor.